src/module_wrap.cc

```diff
@@ -494,6 +493,59 @@ Maybe<uv_file> CheckFile(const URL& search,
   return Just(fd);
 }
 
+PackageConfig emptyPackage = { false, false, "" };
```

Comment:
`static const PackageConfig kEmptyPackage = ...` and since it's only used in GetPackageConfig() move it into that function.